server/pulp/server/db/manage.py

```diff
@@ -294,4 +294,4 @@ def _start_logging():
 
 def _user_input_continue(question):
     reply = str(raw_input(_(question + ' (y/N): '))).lower().strip()
-    return reply[0] == 'y'
+    return not len(reply) == 0 and reply[0] == 'y'
```

Comment:
I thought using conditions would be much faster than try/catch but that is not the case:

```python
In [7]: def test1(s):
   ...:     try:
   ...:         return s[0] == 'y'
   ...:     except IndexError:
   ...:         return False
   ...:     

In [8]: test1('')
Out[8]: False

In [9]: test1('   ')
Out[9]: False

In [10]: def test2(s):
    ...:     return not len(s) == 0 and s[0] == 'y'
    ...: 

In [14]: %timeit "test1('')"
100000000 loops, best of 3: 7.81 ns per loop

In [15]: %timeit "test2('')"
100000000 loops, best of 3: 7.75 ns per loop
```

I will update the code.